src/taskchampion-cpp/corrosion does not contain a CMakeLists.txt file

taskwarrior-3.4.2.tar.gz extracted.

read CONTRIBUTING.md

cmake -S . -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo

but that command fails with:

-- Using Corrosion as a subdirectory
CMake Error at src/taskchampion-cpp/CMakeLists.txt:49 (add_subdirectory):
  The source directory

    src/taskchampion-cpp/corrosion

  does not contain a CMakeLists.txt file.

--- a/doc/devel/contrib/development.md
+++ b/doc/devel/contrib/development.md
@@ -22,6 +22,8 @@ See the general CMake man pages or the [cmake-documentation](https://cmake.org/c
 ```sh
 git clone https://github.com/GothenburgBitFactory/taskwarrior
 cd taskwarrior
+git submodule init
+git submodule update
 cmake -S . -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo
 cmake --build build
 ```

